en/pocket_privacy_policy.md

```diff
@@ -0,0 +1,127 @@
+# Privacy Policy
+
+## *Posted October 11, 2021*
```

Comment:
Other documents don't use an heading level for this, and have a datetime directive. Also, the language is different.

```
*Effective October 11, 2021*
{: datetime="2021-10-11" }